layers/+lang/haskell/packages.el

```diff
@@ -16,26 +16,19 @@
     (company-cabal :requires company)
 
     ;; ghci completion backend
-    (company-ghci
-     :requires company
-     :toggle (eq haskell-completion-backend 'ghci))
+    (company-ghci :requires company)
 
     ;; ghc-mod completion backend
-    (company-ghc
-     :requires company
-     :toggle (eq haskell-completion-backend 'ghc-mod))
+    (company-ghc  :requires company)
     (ghc :toggle (eq haskell-completion-backend 'ghc-mod))
 
     ;; intero completion backend
-    (intero
-     :requires company
-     :toggle (eq haskell-completion-backend 'intero))
+    (intero :requires company)
 
     ;; dante completion backend
     (dante
      :requires company
-     :toggle (and (version<= "25" emacs-version)
-                  (eq haskell-completion-backend 'dante)))
+     :toggle (version<= "25" emacs-version))
```

Comment:
I haven't checked yet, but it looks rubbish that `dante` is available only on old Emacs.

Reply:
I've checked it. And it is no longer the case. Removed all along.